### DISCUSSION TITLE
Show production for groups of countries

### DISCUSSION THREAD

**Discussion by tbertels** [electricitymaps/electricitymaps-contrib#5236] [Ideas]
Since renewable energy isn't always available, the theory is that countries that are able to produce more (e.g. because of a windy day) would export that energy to the other countries.

If we could show the production/installed capacity of a group of countries, we could easily check if that theory holds true.

**Comment by VIKTORVAV99** [comment]
Sorry for taking so long to respond to this.

I think this is a good idea and should be discussed further but since it's not really an issue I'll turn this into a discussion instead.
